src/components/pages/Save/index.tsx

```diff
@@ -67,21 +74,100 @@ const SaveForm: FC<{}> = () => {
   );
 };
 
-export const Save: FC<{}> = () => (
-  <SaveProvider>
-    <FormProvider formId="save">
-      <PageHeader
-        icon={<SaveIcon />}
-        title="SAVE"
-        subtitle="Earn mUSD’s native interest rate"
-      >
-        <P>
-          Deposit your mUSD into the mUSD SAVE contract and start earning
-          interest.
-        </P>
-      </PageHeader>
-      <SaveInfo />
-      <SaveForm />
-    </FormProvider>
-  </SaveProvider>
-);
+const APYStats = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  text-align: center;
+
+  @media (min-width: ${({ theme }) => theme.viewportWidth.xs}) {
+    margin: 0;
+    text-align: right;
+    justify-content: flex-end;
+  }
+`;
+
+const ToggleContainer = styled.div`
+  display: flex;
+`;
+
+const InfoCountUp = styled(CountUp)`
+  font-size: 1.5rem;
+  font-family: 'DM Mono', monospace;
+  color: ${({ theme }) => theme.color.blue};
+`;
+
+const InfoMsg = styled.div`
+  padding-top: 4px;
+  font-size: 12px;
+
+  a {
+    color: ${({ theme }) => theme.color.greyTransparent};
+    border: none;
+    text-decoration: underline;
+  }
+`;
+
+export const Save: FC<{}> = () => {
+  const [activeVersion, setActiveVersion] = useState(CURRENT);
+  const apyForPastWeek = useAverageApyForPastWeek();
+  const versionNumber = activeVersion === CURRENT ? 2 : 1;
+
+  const getVersion = (selection: ToggleSaveSelection): SaveVersion =>
```

Comment:
Quibble: this should either be a static function, or just a variable assigned in the body of the FC; no need to define a function every render.